packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -36,11 +36,17 @@ export const updateBasicConfiguration = createAction(
  */
 export const updateSearchConfiguration = createAction(
   'configuration/updateSearchConfiguration',
-  (payload: {apiBaseUrl?: string; pipeline?: string; searchHub?: string}) =>
+  (payload: {
+    apiBaseUrl?: string;
+    pipeline?: string;
+    searchHub?: string;
+    enableDebug?: boolean;
```

Comment:
Why should this parameter be configurable using initial config and not exclusively through the controller? If someone wants to use the report, won't they initialize the controller anyway?

Reply:
I'm simply reusing the action here, without creating a new separate one.

Reply:
Hummm... enable debug is a bit weird to put inside the `searchConfiguration`, I think.

It's not the same as pipeline or searchHub, which we *want* to guide you to configure correctly initially, and have it global for the duration of the engine lifetime.

Here, I see it more as any other normal query parameter, aka: the same a `q`, `context`, `facets` etc..

So I think we should have a slice for this.